contracts/contracts/strategies/curve/CurveTwoCoinLib.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { ICurveMetaPool } from "./ICurveMetaPool.sol";
+
+library CurveTwoCoinLib {
+    uint256 public constant CURVE_BASE_ASSETS = 2;
+
+    /**
+     * @notice Deposit coins into a Curve pool
+     * @param _pool Address of the Curve pool
+     * @param _amounts List of amounts of coins to deposit
+     * @param _min_mint_amount Minimum amount of LP tokens to mint from the deposit
+     */
+    function add_liquidity(
+        address _pool,
+        uint256[] calldata _amounts,
+        uint256 _min_mint_amount
+    ) external {
+        require(
+            _amounts.length == CURVE_BASE_ASSETS,
+            "Invalid number of amounts"
+        );
+        uint256[CURVE_BASE_ASSETS] memory amount = [_amounts[0], _amounts[1]];
+
+        // slither-disable-next-line unused-return
+        ICurveMetaPool(_pool).add_liquidity(amount, _min_mint_amount);
+    }
+
+    /**
+     * @notice Withdraw coins from the pool
+     * @dev Withdrawal amounts are based on current deposit ratios
+     * @param _pool Address of the Curve pool
+     * @param _burn_amount Quantity of LP tokens to burn in the withdrawal
+     * @param _min_amounts Minimum amounts of underlying coins to receive
+     */
+    function remove_liquidity(
+        address _pool,
+        uint256 _burn_amount,
+        uint256[] calldata _min_amounts
+    ) external {
+        require(
+            _min_amounts.length == CURVE_BASE_ASSETS,
+            "Invalid number of min amounts"
+        );
+        uint256[CURVE_BASE_ASSETS] memory min_amounts = [
+            _min_amounts[0],
+            _min_amounts[1]
+        ];
+
+        // slither-disable-next-line unused-return
+        ICurveMetaPool(_pool).remove_liquidity(_burn_amount, min_amounts);
+    }
+
+    /**
+     * @notice Withdraw coins from the pool
+     * @dev Withdrawal amounts are based on current deposit ratios
+     * @param _pool Address of the Curve pool
+     * @param _burn_amount Quantity of LP tokens to burn in the withdrawal
+     * @param _min_amounts Minimum amounts of underlying coins to receive
+     * @param _receiver Address that receives the withdrawn coins
+     * @param wasTransfer true if the coins were transferred to the receiver, false if not
+     */
+    function remove_liquidity(
+        address _pool,
+        uint256 _burn_amount,
+        uint256[] calldata _min_amounts,
+        address _receiver
```

Comment:
Noticed that CurveThreeCoinLib doesn't have a balanced removal of liquidity where you also specify a `_receiver` address but CurveTwoCoinLib has one.